### DISCUSSION TITLE
LibreSprite version

### DISCUSSION THREAD

**Discussion by Zughy** [LibreSprite/LibreSprite#187] [General]
(testing Discussions, cloning #165 )

The latest pre-release says `v1.2.1-beta`, but LibreSprite says `v1.0-dev`. Which one makes more sense?

The former is simply a continous from the original Aseprite version (it was forked on `1.1.7`), so maybe it can be a nice idea to completely detach from Aseprite as it's (imho) misleading. Also because every time I open LibreSprite, I read "v1.0-dev", so my brain keeps thinking we're on `1.0-dev` anyway.

If `1.0-dev` is kept we could catch two birds with one stone with the roadmap (#160), that would officially start with 1.0. A nice number to say (according to how that issue is going) we now supports Mac and that we got rid of Allegro 4. I mean, it's a nice place where to start

**Comment by maujin111** [comment]
The actual version should be called v1.0-beta and when the job with sdl2, JavaScript scripting and macOs support is done. Release the stable one

**Reply by Steph155** [reply]
Greetings @maujin111 
I want to inform you that the official stable v1.0 Windows version of LibreSprite has much lower framerate compared to the earlier 2020 v1.0 dev version. Also, I don't know why but, there are new libs in LibreSprite's folder but in the older version, there were pretty much none.
